sorc/chgres_cube.fd/write_data.F90

```diff
@@ -538,7 +538,7 @@ subroutine write_fv3_atm_bndy_data_netcdf(localpet)
    elseif (trim(input_type) == "gaussian_netcdf") then
```

Comment:
@hu5970, I don't believe these changes are still required based on Chan-Hoo's PR to ufs-weather-model [here](https://github.com/ufs-community/ufs-weather-model/pull/567).  Can we test without these changes first?  If we determine that the hydrometeors are present at initialization with the GRIB2 flag, then it's working as expected.

Reply:
@JeffBeck-NOAA Yes, I can test if this one is still needed or not because you did mentioned this is not required in new code base. Any other code we need to change. I can update the code and then test it in dev2, which is using grib2 files from GFS.

Reply:
OK, that sounds good.  I think the only change we needed was related to the grib2 fields that is in this PR.  @LarissaReames-NOAA is working on implementing the grib2 field fix in the authoritative repo, and a PR will likely be opened soon.